test/mastodon-tl-tests.el

```diff
@@ -1,4 +1,5 @@
 (require 'el-mock)
+(require 'cl-macs)
```

Comment:
Dependent on `cl-lib`. I'm not sure what the policy is on including `cl-lib` in tests though, will have to be checked by @jdenen